frb_dart/lib/src/ffi/io.dart

```diff
@@ -19,6 +19,24 @@ extension StoreDartPostCObjectExt on FlutterRustBridgeWireBase {
   }
 }
 
+class DartApiDl {
+  static int? _initCode;
+
+  static void initApi(ffi.DynamicLibrary dylib) {
+    var code = _initCode ?? _internal(dylib);
+    if (code != 0) {
+      throw 'Failed to initialize Dart API. Code: $_initCode';
+    }
+  }
+
+  static int _internal(ffi.DynamicLibrary dylib) {
+    _initCode = dylib.lookupFunction<ffi.IntPtr Function(ffi.Pointer<ffi.Void>),
```

Comment:
nit: Is it possible we use automatically generated signatures instead of manually writing down the lookupFunction?